src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -54,6 +54,11 @@
 from psyclone.psyir.nodes.directive import StandaloneDirective, \
     RegionDirective
 from psyclone.psyir.nodes.loop import Loop
+from psyclone.psyir.nodes.literal import Literal
+from psyclone.psyir.nodes.omp_clauses import GrainsizeClause, NowaitClause,\
+    NogroupClause, NumTasksClause
+from psyclone.psyir.nodes.schedule import Schedule
+from psyclone.psyir.symbols import INTEGER_TYPE
```

Comment:
Again there is a lot of "six" use in this file than may be possible to delete now, but I am not entirely sure.